internal/cli/atlas/users/invite.go

```diff
@@ -94,12 +94,12 @@ func (opts *InviteOpts) Run() error {
 
 const keyParts = 2
 
-func (opts *InviteOpts) createRoles() ([]atlasv2.RoleAssignment, error) {
+func (opts *InviteOpts) createRoles() ([]atlasv2.CloudRoleAssignment, error) {
 	if !config.IsCloud() {
 		return nil, nil
 	}
 
-	atlasRoles := make([]atlasv2.RoleAssignment, len(opts.orgRoles)+len(opts.projectRoles))
```

Comment:
Looks like we have two objects RoleAssignment and now CloudRoleAssignment (they were overriding each other).